climpred/bootstrap.py

```diff
@@ -372,7 +451,7 @@ def bootstrap_compute(
     sig=95,
     bootstrap=500,
     pers_sig=None,
-    compute=compute_hindcast,
+    compute=verify_hindcast,
```

Comment:
I see why you want to rename `compute` to `verify`. but this inflates this PR and thus makes the review much harder.

Reply:
I find the name `compute_hindcast` very vague, since we're always computing something with the hindcast. But I agree. I'll tag you for a review once it's done. There's some more things I want to work out. 

Refactoring PRs are a pain to look through so I'll have a changelog for you of the things that were changed.